### DISCUSSION TITLE
odd warning for nyholm/psr7

### DISCUSSION THREAD

**Discussion by tacman** [ToshY/BunnyNet-PHP#141] [General]
>   - Skipping recipe for nyholm/psr7: all versions of the recipe conflict with your package versions.

```bash
~/g/museado/museado$ composer why nyholm/psr7
php-http/discovery 1.19.4 conflicts nyholm/psr7 (<1.0)   
toshy/bunnynet-php 4.5.0  requires  nyholm/psr7 (^1.6.1)
 
~/g/museado/museado$ composer why php-http/discovery
knplabs/github-api                v3.14.1 requires  php-http/discovery (^1.12) 
meilisearch/meilisearch-php       v1.10.0 requires  php-http/discovery (^1.7)  
php-http/multipart-stream-builder 1.4.2   requires  php-http/discovery (^1.15) 
symfony/http-client               v7.1.4  conflicts php-http/discovery (<1.15) 
symfony/psr-http-message-bridge   v7.1.4  conflicts php-http/discovery (<1.15) 
```

I don't think this causes any problems, and all the versions _look_ okay to me.  But it seems odd.

**Comment by ToshY** [comment]
The warning originates from [Symfony Flex](https://github.com/symfony/flex) for the recipe of [`nyholm/psr7`](https://github.com/symfony/recipes/tree/main/nyholm/psr7/1.0).

The recipe [`manifest.json`](https://github.com/symfony/recipes/blob/main/nyholm/psr7/1.0/manifest.json) denotes conflict with `php-http/discovery": ">=1.18"`. You are using `1.19.4` according to the above, hence the warning. The conflict was added in PR https://github.com/symfony/recipes/pull/1198.

As this seems related to recipes and not this library, I will close this discussion.

**Reply by tacman** [reply]
Thanks!
